### PR TITLE
update docs for the new lsc network update

### PR DESCRIPTION
**Pull request by kashishshah**
## Description

---

## Reviewers

- @Lagrange-Labs/lsc-core-dev

### CODE REVIEW

File: docs/state-committees/run-node/configuration.md [hunk 1]
```diff
@@ -25,22 +25,50 @@ The `config.toml` file in the repository contains various fields that should be
...
 - `ConcurrentFetchers`: This parameter can be used to control the number of parallel fetchers that are responsible to fetch a block. Default value is `8`.
 
-- `OperatorKeystorePath` = EigenLayer operator keystore file path
+- `SignerServerURL`: This is the gRPC server URL for signer for secure key management.
 
-- `OperatorKeystorePasswordPath` = EigenLayer operator keystore password file path
+- `OperatorAddress`: EigenLayer operator address
 
-- `SignerECDSAKeystorePath` = ECDSA signer keystore file path
+- `AccountID`: Account ID for setting up signer for your keys. The operator can choose the account id of their choice.
```
Comment (mastereng12) [inline docs/state-committees/run-node/configuration.md:32]: what is `AccountID`?

Comment (kashishshah) [reply docs/state-committees/run-node/configuration.md:32]: https://github.com/Lagrange-Labs/lsc-client-cli/blob/develop/config_signer.toml#L11

File: docs/state-committees/run-node/configuration.md [hunk 2]
```diff
@@ -25,22 +25,50 @@ The `config.toml` file in the repository contains various fields that should be
...
 
-- `SignerECDSAKeystorePasswordPath` = ECDSA signer keystore password file path
+- `KeyType`: `BN254` or `ECDSA`
 
-- `BLSKeystorePath` = BLS keystore file path
+- `PrivateKeyPath`: File path for the ECDSA or BLS private key with extension `.key`
 
-- `BLSKeystorePasswordPath` = BLS keystore password file path
+- `PasswordKeyPath`: Password file for the private key
```
[the remark this answers is not in the document]
Comment (kashishshah) [reply docs/state-committees/run-node/configuration.md:38]: Isn't that `KeyType`?  `PrivateKeyPath` is the file path for BN254 or ECDSA key stored in keystore and `PassworKeyPath` is the password for the keystore

File: docs/state-committees/run-node/deploy-attestation-node.md [hunk 3]
```diff
@@ -9,8 +9,22 @@ Once an operator has successfully registered to Lagrange State Committees AVS an
 
 ## Deploy using CLI
 
-1. Run `generate-config` command to generate a config specific to the chain and network for which you want to run an attestation node.
-2. Run `deploy` command to deploy the docker container of your attestation node.
+In the latest version of the CLI (`v1.1.x`), we have introduced the feature to configure TLS for gRPC communication of the attestation nodes and also a remote Signer component for secure key management. If the operator doesn't want to configure these features, please use the CLI version `v1.0.x` and start from step #3.
```
[the remark this answers is not in the document]
Comment (kashishshah) [reply docs/state-committees/run-node/deploy-attestation-node.md:12]: Sure, I will change this

**Pull request merged**